### PR TITLE
Adjusted my bio.

### PR DESCRIPTION
**Pull request by charettes**

### CODE REVIEW

File: docs/internals/team.txt [hunk 1]
```diff
@@ -315,20 +315,18 @@ Jeremy Dunck
     .. _shoutcaster: http://twitch.tv/vlogalonstar/
 
 `Simon Charette`_
-    Simon is a mathematics student who discovered Django while searching for a
+    Simon use to study mathematics and discovered Django while searching for a
```
Comment (MarkusH) [inline docs/internals/team.txt]: use_d_ I think ;)

Comment (charettes) [inline docs/internals/team.txt]: doh, thanks Markus.

**Comment by timgraham** [comment]
As described in https://github.com/django/djangoproject.com/issues/720, I hoping to remove this page from the documentation so we don't need to add commits to the Django repo to update this data. Unfortunately, I'm lacking a review for https://github.com/django/djangoproject.com/pull/745. Maybe you could help out there? ;-)

**Comment by timgraham** [comment]
You should now be able to add your bio in the djangoproject.com admin.

**Pull request closed without merging**